crates/forge-runner/src/gas/report.rs

```diff
@@ -117,3 +121,49 @@ fn get_selector(contracts_data: &ContractsDataStore, selector: EntryPointSelecto
         .0
         .clone()
 }
+
+impl Display for ReportData {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        for (name, contract_info) in &self.0 {
+            if !contract_info.functions.is_empty() {
+                let table = format_table_output(contract_info, name);
+                writeln!(f, "\n{table}")?;
```

Comment:
In what case can there be contract without functions? In case it wasn't actually called in tests or some other ones as well?